src/WebScrap/tests/WebScrap.Tests/Demo.cs

```diff
@@ -53,14 +53,14 @@ public void Demo1()
 
     [Test]
     [Explicit]
-    public async Task Demo_Url()
+    public void Demo_Url()
     {
         var request = "https://www.gpucheck.com/gpu-benchmark-graphics-card-comparison-chart";
 
         // Download the html:
         using var client = new HttpClient();
-        using var response = await client.GetAsync(request);
-        var html = await response.Content.ReadAsStringAsync();
+        using var response = client.GetAsync(request).Result;
+        var html = response.Content.ReadAsStringAsync().Result;
 
         // Run the WebScrapper:
         var css = "table";
```

Comment:
I dont like these tests. They are more like for debugging. Maybe move it to separate folder with console app for a playground?

Reply:
It's still a part of the repository.
Should I create a project, like a console, add reference to API library? Basically duplicate all the stuff? Nah, wont do.